application/analyst/services/chapter_indexing_service.py

```
@@ -49,7 +49,9 @@ def _get_collection_name(self, novel_id: str) -> str:
         Returns:
             collection 名称，格式为 novel_{novel_id}_chunks
         """
-        return f"novel_{novel_id}_chunks"
+        # novel_id 可能包含 "novel-" 前缀，需要去掉避免重复
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)